src/lib/components/ILSMenu/ILSMenu.js

```diff
@@ -45,12 +45,12 @@ class ILSMenu extends Component {
 
     return (
       <>
-        <Media greaterThanOrEqual="tablet">
+        <Media greaterThanOrEqual="computer">
```

Comment:
how does it look on the tablet after this change ?

Reply:
On an iPad (768x1024) it looks like this :
![image](https://user-images.githubusercontent.com/15194802/109309261-5179a280-7843-11eb-8375-f8f50fb9d743.png)